ddns-ipv64.xml

```diff
@@ -0,0 +1,27 @@
+  <Category>Tools: Network:DNS</Category>
+  <Icon>https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQasFPd0QKbbYpPvuUG0EXhoVFF-Kw2XcdqDg&amp;amp;s</Icon>
+  <Requires>List of all Parameters that can be configured https://github.com/alcapone1933/docker-ddns-ipv64?tab=readme-ov-file#env-parameter</Requires>
+  <Config Name="Timezone" Target="TZ" Default="Europe/Berlin" Mode="" Description="Timezone" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Polling frequency IP address" Target="CRON_TIME" Default="*/15 * * * *" Mode="" Description="Polling frequency of the current IP address" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Polling frequenz Domain" Target="CRON_TIME_DIG" Default="*/30 * * * *" Mode="" Description="Polling frequenz for the domain (dig DOMAIN_IPV64 A)" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain(s) ipv64" Target="DOMAIN_IPV64" Default="" Mode="" Description="YOUR DOMAINS: e.g. your-domain.ipv64.net, your-domain.ipv64.de" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain Key" Target="DOMAIN_KEY" Default="" Mode="" Description="DOMAIN KEY: YOUR DOMAIN KEY or DynDNS update hash can be found at https://ipv64.net/dyndns.php" Type="Variable" Display="always" Required="true" Mask="true"/>
+  <Config Name="Domain Präfix yes|no" Target="DOMAIN_PRAEFIX_YES" Default="yes|no" Mode="" Description="DOMAIN PRAEFIX YES: This means that the domain PRAEFIX is actively used" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Domain Präfix" Target="DOMAIN_PRAEFIX" Default="" Mode="" Description="YOUR DOMAIN PRAEFIX (subdomain) : &#x26A0;&#xFE0F; Use only one prefix &#x26A0;&#xFE0F; e.g. ddns" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="IP check" Target="IP_CHECK" Default="yes|no" Mode="" Description="IP CHECK: The IP address of the domain is checked" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Shoutrrr URL" Target="SHOUTRRR_URL" Default="" Mode="" Description="SHOUTRRR URL: Your Shoutrrr URL as notification service e.g. ( gotify,discord,telegram,email)" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Shoutrrr URL Skip Test" Target="SHOUTRRR_SKIP_TEST" Default="" Mode="" Description="SHOUTRRR_SKIP_TEST: No test message is sent when the container is started" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Name Server" Target="NAME_SERVER" Default="ns1.ipv64.net" Mode="" Description="NAME_SERVER: : The name server to check the IP address of your domain" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Location logs and script" Target="/data" Default="/mnt/user/appdata/ddns-ipv64" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false"/>
```

Comment:
```suggestion
  <Config Name="Location logs and script" Target="/data" Default="/mnt/user/appdata/ddns-ipv64" Mode="rw" Description="" Type="Path" Display="advanced-hide" Required="false" Mask="false">/mnt/user/appdata/ddns-ipv64</Config>
```

Config paths for containers stored in the "appdata" folder are typically put in the "advanced" section, as they don't require immediate set-up by the user (given they have a valid default). Marking is as "advanced-hide" will prevent users from being able to delete this config value from the GUI (force them to set it for their own sake).